src/js/components/PageHeader/PageHeader.js

```diff
@@ -8,13 +8,19 @@ import { Grid } from '../Grid';
 import { Paragraph } from '../Paragraph';
 import { ResponsiveContext } from '../../contexts/ResponsiveContext';
 
+const sizeStyle = (size, style, theme) =>
```

Comment:
Minor, but for some reason I don't immediately associate `title`, `subtitle`, `pad` with `style`. Something like property, attribute, item, aspect, feature, etc. might feel more fitting.

```suggestion
const sizeStyle = (size, feature, theme) =>
```

Reply:
Done.